### PR TITLE
Logspout example

### PR DESCRIPTION
**Pull request by christianacca**
See individual commits for details

### CODE REVIEW

**Comment by FantasticFiasco** [comment]
Hi and welcome to the repository!

Logspout looks awesome, I haven't seen it before. I think is should be presented as another solution to sending log events to the Elastic Stack, and I think the solution is important enough to grant it its own repo. Would you accept becoming a collaborator on this new repo, and move your commits there?

**Comment by christianacca** [comment]
Great, yes set me up :-)

File: serilog/Program.cs [hunk 1]
```diff
@@ -26,11 +22,37 @@ static void Main()
...
+            if (!useLogspout)
+            {
+                // log direct to logstash
+                return new LoggerConfiguration()
+                    .WriteTo.DurableHttpUsingFileSizeRolledBuffers(
+                        requestUri: "http://logstash:31311",
+                        textFormatter: new ElasticsearchJsonFormatter(),
+                        batchFormatter: new ArrayBatchFormatter())
+                    .WriteTo.Console();
+            }
```
Comment (christianacca) [inline serilog/Program.cs]: So if we're proposing to create a new repo showing logspout as the method for sending to logstash, then this sink should be removed, and the conditional logic for selecting it should be removed - right?

**Comment by christianacca** [comment]
Just added a comment to the PR: https://github.com/FantasticFiasco/serilog-sinks-http-sample-elastic-stack/pull/41/files#r294380826

Have a look, to confirm we're thinking the same thing

File: serilog/Program.cs (hunk 1 above)
Comment (FantasticFiasco) [reply serilog/Program.cs]: Correct, that's the reason I think it should be a new repository and not cause confusion in a sample of Serilog.Sinks.Http.

I'll get you the repo with configured Elastic Stack in a few hours, or a empty repository for you to fill right now if you are up for it.

**Comment by FantasticFiasco** [comment]
You are now invited to the new repository. Please let me know if you are unable to push changes.

**Comment by FantasticFiasco** [comment]
The Logspout sample application is now linked from [Serilog.Sinks.Http](https://github.com/FantasticFiasco/serilog-sinks-http).

**Pull request closed without merging**